src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANDOM = new Random().nextInt(5);
+    private static final String CIRCLE = "Circle";
+    private static final String ISOSCELES_TRAPEZOID = "IsoscelesTrapezoid";
+    private static final String RECTANGLE = "Rectangle";
+    private static final String RIGHT_TRIANGLE = "RightTriangle";
+    private static final String SQUARE = "Square";
+    private static final String[] nameFigure = new String[]
+            {CIRCLE, ISOSCELES_TRAPEZOID, RECTANGLE, RIGHT_TRIANGLE, SQUARE};
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        switch (nameFigure[RANDOM]) {
+            case CIRCLE:
+                Figure f1 = new Circle(
```

Comment:
```suggestion
                Figure circle = new Circle(
```

let's use informative variable use. Numbers in variable names it's bad practice

Reply:
I rewrote it so:

Figure figure;
        switch (nameFigure[RANDOM]) {
            case CIRCLE:
                figure = new Circle(